x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -224,6 +231,9 @@ public static EqlSearchRequest fromXContent(XContentParser parser) {
             (p, c) -> TimeValue.parseTimeValue(p.text(), KEY_KEEP_ALIVE), KEEP_ALIVE, ObjectParser.ValueType.VALUE);
         parser.declareBoolean(EqlSearchRequest::keepOnCompletion, KEEP_ON_COMPLETION);
         parser.declareBoolean(EqlSearchRequest::isCaseSensitive, CASE_SENSITIVE);
+        parser.declareString((request, order) -> {
```

Comment:
It's better to have a setter with a string and then create the Enum inside that method (essentially move there the `fromString`).